### PR TITLE
 add type detect in globby mode

### PR DESCRIPTION
**Pull request by agileago**

### CODE REVIEW

**Comment by SimenB** [comment]
Wow, that was noisy. I've disabled the bot.

@agileago I'd love to see this added, why did you close?

**Comment by agileago** [comment]
so many error ! i don't how to resolve it . so close it. i just have a little experience to create `PR`. sorry.

**Comment by SimenB** [comment]
Ignore the bot, it was being stupid

**Comment by SimenB** [comment]
What you can so is run `yarn && yarn lint --fix`, which should fix the current errors.

After that we need a new test. The current tests should be pretty easy to copy and tweak, but I can guide you through it if you want 🙂

**Comment by agileago** [comment]
thanks. i do it

**Comment by agileago** [comment]
thanks. i do it

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ new AddAssetHtmlPlugin([
   { filepath: require.resolve('./some-file') }, 
   { filepath: require.resolve('./some-other-file') },
   // Glob to match all of the dll file
-  { filepath: require.resolve('./**/*.dll.js') },
+  { filepath: './**/*.dll.js' },
```
Comment (SimenB) [inline README.md]: good catch!

**Comment by SimenB** [comment]
Can you add a test and update the changelog?

Also write somewhere in the readme that the file type is inferred.

**Comment by agileago** [comment]
I try it.

**Comment by agileago** [comment]
i don't know how to write this test rightly. maybe you need to correct it. thanks.

**Pull request closed without merging**